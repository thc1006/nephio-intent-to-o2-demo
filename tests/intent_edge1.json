{
  "intentId": "test-001",
  "serviceType": "enhanced-mobile-broadband",
  "targetSite": "edge1",
  "resourceProfile": "standard",
  "sla": {
    "availability": 99.99,
    "latency": 10,
    "throughput": 1000
  }
}