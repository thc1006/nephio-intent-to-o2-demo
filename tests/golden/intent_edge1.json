{
  "intentExpectationId": "golden-edge1-001",
  "intentExpectationType": "NetworkSliceIntent",
  "targetSite": "edge1",
  "intent": {
    "serviceName": "golden-service-edge1",
    "serviceType": "eMBB",
    "networkSlice": {
      "sliceId": "golden-slice-edge1-001",
      "sliceType": "eMBB",
      "maxNumberofUEs": 10000,
      "maxNumberofConnections": 50000
    },
    "qos": {
      "downlinkThroughput": "1Gbps",
      "uplinkThroughput": "100Mbps",
      "latency": "20ms",
      "reliability": "99.9%"
    },
    "coverageArea": {
      "site": "edge1",
      "region": "us-west",
      "zone": "zone-1"
    },
    "lifecycle": {
      "startTime": "2025-01-01T00:00:00Z",
      "endTime": "2025-12-31T23:59:59Z"
    }
  }
}